opal/datatype/opal_convertor.c

```diff
@@ -49,19 +49,6 @@
 extern int opal_convertor_create_stack_with_pos_general( opal_convertor_t* convertor,
                                                          int starting_point, const int* sizes );
 
-static void opal_convertor_construct( opal_convertor_t* convertor )
-{
-    convertor->pStack         = convertor->static_stack;
-    convertor->stack_size     = DT_STATIC_STACK_SIZE;
-    convertor->partial_length = 0;
-    convertor->remoteArch     = opal_local_arch;
-    convertor->flags          = OPAL_DATATYPE_FLAG_NO_GAPS | CONVERTOR_COMPLETED;
-#if OPAL_CUDA_SUPPORT
-    convertor->cbmemcpy       = &opal_cuda_memcpy;
-#endif
-}
-
-
```

Comment:
@bosilca - Thanks for reviewing.

Yeah, OBJ_CONSTRUCT and OBJ_DESTRUCT seems to be a better fit (I was not aware of these). I had noticed OBJ_NEW, but that will allocate it.
Let me update it based on your suggestion. Thanks